x-pack/plugins/fleet/server/types/files/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export type FILE_STATUS = 'AWAITING_UPLOAD' | 'UPLOADING' | 'READY' | 'UPLOAD_ERROR' | 'DELETED';
```

Comment:
Consider using the `FIleStatus` type from the Files plugin here instead of hard-coding it:

```typescript=
import type { FileStatus } from '@kbn/files-plugin/common/types';
```
 
Ref: 
https://github.com/elastic/kibana/blob/1ed2ec8e57ab5da100988b00c697e2d6d6d615e9/src/plugins/files/common/types.ts#L28-L37

Reply:
I was originally using this but moved away since I didn't want to add a plugin dependency just for this. If that isn't a big deal, can add it back in.

Reply:
we will need a plugin dependency for the request diagnostics feature anyway